Modules/TestQuestionPool/classes/class.assClozeGapCombination.php

```diff
@@ -185,7 +185,7 @@ public function getGapsWhichAreUsedInCombination($question_id)
 	{
 		global $ilDB;
 
-		$result = $ilDB->queryF('SELECT gap_fi, combination_id FROM `qpl_a_cloze_combi_res` WHERE question_fi = %s GROUP BY gap_fi, combination_id',
+		$result = $ilDB->queryF('SELECT gap_fi, combination_id FROM ' . $ilDB->quoteIdentifier('qpl_a_cloze_combi_res') . ' WHERE question_fi = %s GROUP BY gap_fi, combination_id',
```

Comment:
@bheyser ok for you? quoting the table-name isn't really needed here.

Reply:
this is okay for me :+1: